cirq-core/cirq/ops/global_phase_op.py

```diff
@@ -17,28 +17,59 @@
 import numpy as np
 
 from cirq import value, protocols
-from cirq.ops import raw_types
+from cirq._compat import deprecated, deprecated_class
+from cirq.ops import gate_operation, raw_types
 
 if TYPE_CHECKING:
     import cirq
 
 
 @value.value_equality(approximate=True)
-class GlobalPhaseOperation(raw_types.Operation):
+@deprecated_class(deadline='v0.15', fix='Use global_phase_operation')
```

Comment:
I expect this to be a very widely used operation and we are already at v0.14dev. Let's put the deadline to be 'v0.16'.

Also, "Use cirq.global_phase_operation"